fury/animation/animation.py

```diff
@@ -1,1446 +1,1454 @@
-from collections import defaultdict
```

Comment:
Why do you comment all this file ? it just needs few update /few comments.... 

I do not understand. most of it do not use vtk...